project-api-tests/src/test/java/org/ballerina/projectapi/CentralTestUtils.java

```diff
@@ -199,4 +229,148 @@ static Path getBalaPath(Path projectPath, String org, String pkgName, String pla
         return projectPath.resolve("target").resolve("bala")
                 .resolve(org + "-" + pkgName + "-" + platform + "-" + version + ".bala");
     }
+
+    /**
+     * Check if package already available on central.
+     *
+     * @param pkg package
+     * @param tempWorkspaceDirectory Path to workspace
+     * @param envVariables Environmental variables
+     * @return whether the package is available on central
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    static boolean isPkgAvailableInCentral(String pkg, Path tempWorkspaceDirectory,
+                                           Map<String, String> envVariables) throws IOException, InterruptedException {
+        String buildOutput = searchPackageDetails(pkg, tempWorkspaceDirectory, envVariables);
+        return !buildOutput.contains("no modules found");
+    }
+
+    /**
+     * Returns true if the provided version is available on central.
+     *
+     * @param pkg package
+     * @param tempWorkspaceDirectory Path to workspace
+     * @param envVariables Environmental variables
+     * @param version version to check availability
+     * @return
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    static boolean isPkgVersionAvailableInCentral(String pkg, Path tempWorkspaceDirectory,
+                                                  Map<String, String> envVariables, String version)
+            throws IOException, InterruptedException {
+        String buildOutput = searchPackageDetails(pkg, tempWorkspaceDirectory, envVariables);
+        return (!buildOutput.contains("no modules found") && buildOutput.contains(version));
+    }
+
+    /**
+     * Search for the given package on Central.
+     *
+     * @param pkg package
+     * @param tempWorkspaceDirectory Path to workspace
+     * @param envVariables Environmental variables
+     * @return
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    private static String searchPackageDetails(String pkg, Path tempWorkspaceDirectory,
+                                           Map<String, String> envVariables) throws IOException, InterruptedException {
+        Process search = executeSearchCommand(DISTRIBUTION_FILE_NAME,
+                tempWorkspaceDirectory,
+                new LinkedList<>(Collections.singletonList(pkg)),
+                envVariables);
+        String buildErrors = getString(search.getErrorStream());
+        if (!buildErrors.isEmpty()) {
+            Assert.fail(OUTPUT_CONTAIN_ERRORS + buildErrors);
+        }
+        String buildOutput = getString(search.getInputStream());
```

Comment:
```suggestion
        return getString(search.getInputStream());
```
We can directly return right?

Reply:
Fixed with 7189e44c00191cfb0e74d468ffcc2b266bdc40c7